sixtyfps_runtime/corelib/graphics/image.rs

```diff
@@ -310,6 +310,36 @@ pub struct LoadImageError(());
 /// let image = Image::from_rgba8(buffer);
 /// ```
 ///
+/// A popular software (CPU) rendering library in Rust is tiny-skia. The following example shows
+/// how to use tiny-skia to render into a [`SharedPixelBuffer`]:
+/// ```no_run
```

Comment:
why no_run ?
I think this can as well be ran
The previous example can't be run because it loads a file that doesn't exist

Reply:
Good point! Bad habit, I’ll change this to allow running.